.gitlab/docker_common/publish_job_templates.yml

```diff
@@ -18,4 +18,6 @@
     - ECR_RELEASE_SUFFIX="${CI_COMMIT_TAG+-release}"
     - IMG_VARIABLES="$(sed -E "s#(${SRC_AGENT}|${SRC_DSD}|${SRC_DCA}|${SRC_CWS_INSTRUMENTATION})#\1${ECR_RELEASE_SUFFIX}#g" <<<"$IMG_VARIABLES")"
     - IMG_SOURCES="$(sed -E "s#(${SRC_AGENT}|${SRC_DSD}|${SRC_DCA}|${SRC_CWS_INSTRUMENTATION})#\1${ECR_RELEASE_SUFFIX}#g" <<<"$IMG_SOURCES")"
-    - inv pipeline.trigger-child-pipeline --project-name "DataDog/public-images" --git-ref "main" --variables "IMG_VARIABLES,IMG_REGISTRIES,IMG_SOURCES,IMG_DESTINATIONS,IMG_SIGNING"
+    - >
+      inv pipeline.trigger-child-pipeline --project-name "DataDog/public-images" --git-ref "main"
+      --variables "IMG_VARIABLES,IMG_REGISTRIES,IMG_SOURCES,IMG_DESTINATIONS,IMG_SIGNING,APPS,BAZEL_TARGET,DDR,DDR_WORKFLOW_ID,TARGET_ENV,DYNAMIC_BUILD_RENDER_TARGET_FORWARD_PARAMETERS"
```

Comment:
maybe we could use several lines for the variable list, would improve readability wdyt? (Same comment applies everywhere, sorry in advance for that if you decide to change)

Reply:
You're right. I'm always a bit scared of splitting lines in yaml but I think the new version should work :crossed_fingers: 

Test pipeline triggered at: https://gitlab.ddbuild.io/DataDog/datadog-agent/-/pipelines/27302495